fenix/app/src/main/java/org/mozilla/fenix/components/TrackingProtectionPolicyFactory.kt

```diff
@@ -106,9 +106,9 @@ class TrackingProtectionPolicyFactory(
     }
 }
 
+@Suppress("MaxLineLength")
 @VisibleForTesting
-internal fun TrackingProtectionPolicyForSessionTypes.applyTCPIfNeeded(settings: Settings):
-    TrackingProtectionPolicyForSessionTypes {
+internal fun TrackingProtectionPolicyForSessionTypes.applyTCPIfNeeded(settings: Settings): TrackingProtectionPolicyForSessionTypes {
```

Comment:
Could we undo this change to not hit the `MaxLineLength`?

Reply:
@gabrielluong The alternative is suppressing a `Single space expected between colon and return type (standard:function-return-type-spacing)` error as I don't see a good way of appeasing that rule without exceeding the maximum line length. IMO, I found the line length suppression to be the reasonable trade-off for better readability and consistency throughout the file. Let me know, though!

Reply:
Sounds reasonable. I guess this is more of an unfortunate issue with the how we have named things here.